src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/BulkPrintDocumentService.java

```diff
@@ -43,39 +43,52 @@ public void validateEncryptionOnUploadedDocument(CaseDocument caseDocument,
         if (caseDocument != null) {
             String documentFilename = caseDocument.getDocumentFilename();
             log.info("checking encryption for file {} for Case ID: {}", documentFilename, caseId);
-            byte[] pdfBytes;
-            if (documentFilename.toLowerCase().endsWith(".doc") || documentFilename.toLowerCase().endsWith(".docx")) {
-                Document document = Document.builder().url(caseDocument.getDocumentUrl())
-                    .binaryUrl(caseDocument.getDocumentBinaryUrl())
-                    .fileName(caseDocument.getDocumentFilename())
-                    .build();
 
-                pdfBytes = documentConversionService.convertDocumentToPdf(document, auth);
-            } else {
-                pdfBytes = service.download(caseDocument.getDocumentBinaryUrl(), auth);
+            if (documentFilename.toLowerCase().endsWith(".doc") || documentFilename.toLowerCase().endsWith(".docx")) {
```

Comment:
you may get code conflict soon. Consider putting the change to your branch?!

https://github.com/hmcts/finrem-case-orchestration-service/pull/2160/files#diff-b6b7ce98d00a7b2c510a4ecb9478c3fabc0ade381296191a22e5ba63dcdf07b7R50